code/modules/maps/planet_types/lore/tauceti.dm

```diff
@@ -133,3 +133,59 @@
 
 /obj/effect/overmap/visitable/sector/exoplanet/barren/asteroid/dumas/generate_ground_survey_result()
 	ground_survey_result = "<br>No notable deposits underground"
+
+// --------------------------------- Biesel
+/obj/effect/overmap/visitable/sector/exoplanet/biesel
+	name = "Biesel"
+	desc = "The third closest planet to Tau Ceti's star, Biesel is an Earth-like planet that benefits from a temperate climate and breathable atmosphere. It is the capital planet of the Republic of Biesel."
+	icon_state = "globe2"
+	color = "#5B8958"
+	planetary_area = /area/exoplanet/grass
+	initial_weather_state = /singleton/state/weather/calm/snow_planet
+	scanimage = "biesel.png"
+	massvolume = "0.95~/1.1"
+	surfacegravity = "0.99"
+	charted = "Charted 2147CE, Sol Alliance Department of Colonization."
+	geology = "Low-energy tectonic heat signature, minimal surface disruption"
+	weather = "Global full-atmosphere hydrological weather system."
+	surfacewater = "Majority potable, 75% surface water. Significant tidal forces from natural satellite"
+	rock_colors = list(COLOR_BROWN)
+	flora_diversity = 0
+	has_trees = FALSE
```

Comment:
i'm not exactly sure what flora_diversity and has_trees does, but i figure Biesel would have some. if you have a reason for it being set like this (like the biesel theme handling it) just comment that here please